config/prod/deploy.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
```

Comment:
On second thought, do you think we should try to keep the dev/prod deployment scripts the same and handle any differences with a cmd line arg? to reduce testing fanout?

Reply:
ack. SG. As discussed offline, I will combine them in a follow up PR.